src/arrayMethodJoin.js

```diff
@@ -6,10 +6,10 @@
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
     let output = '';
-    let correctSeparator = separator;
+    let validSeparator = separator;
 
     if (separator === undefined) {
-      correctSeparator = ',';
+      validSeparator = ',';
```

Comment:
Ліпше задай кому, у якості стандартного значення. 
Три стрічки коду зекономиш _)
```suggestion
      [].__proto__.join2 = function(separator = ',') {
```

Reply:
Ось тут, в сенсі.
![image](https://user-images.githubusercontent.com/63505873/83254012-ac983f00-a1b6-11ea-89e7-97a3bdbebe76.png)

Reply:
круто! дякую, ще й скоротив одну змінну